intermediate-algorithm-scripting-yahtzee-game/index.html

```diff
@@ -1,104 +1,126 @@
+          <strong>Rolls:</strong> <span id="current-round-rolls">0</span> |
+          <strong>Round:</strong> <span id="current-round">1</span>
+        </p>
 
-  <hr>
+        <div id="score-options">
+          <div>
+            <input
+              type="radio"
+              name="score-options"
+              id="three"
+              value="three-of-a-kind"
+              disabled
+            />
+            <label for="three">Three of a kind<span></span></label>
+          </div>
+          <div>
+            <input
+              type="radio"
+              name="score-options"
+              id="four"
+              value="four-of-a-kind"
+              disabled
+            />
+            <label for="four">Four of a kind<span></span></label>
+          </div>
+          <div>
+            <input
+              type="radio"
+              name="score-options"
+              id="full"
+              value="full-house"
+              disabled
+            />
+            <label for="full">Full house<span></span></label>
+          </div>
+          <div>
+            <input
+              type="radio"
+              name="score-options"
+              id="small"
+              value="small-straight"
+              disabled
+            />
+            <label for="small">Small straight<span></span></label>
+          </div>
+          <div>
+            <input
+              type="radio"
+              name="score-options"
+              id="large"
+              value="large-straight"
+              disabled
+            />
+            <label for="large">Large straight<span></span></label>
+          </div>
+          <div>
+            <input
+              type="radio"
+              name="score-options"
+              id="yahtzee"
+              value="yahtzee"
+              disabled
+            />
+            <label for="yahtzee">Yahtzee<span></span></label>
+          </div>
+          <div>
+            <input
+              type="radio"
+              name="score-options"
+              id="chance"
+              value="chance"
+              disabled
+            />
+            <label for="chance">Chance<span></span></label>
+          </div>
+          <div>
+            <input
+              type="radio"
+              name="score-options"
+              id="none"
+              value="none"
+              disabled
+            />
+            <label for="none">None of the above<span></span></label>
+          </div>
+        </div>
 
-  <main>
-    <div id="game">
-      <div id="dice">
-        <div class="die selected"></div>
-        <div class="die selected"></div>
-        <div class="die selected"></div>
-        <div class="die selected"></div>
-        <div class="die selected"></div>
+        <button class="btn" id="keep-score-btn" disabled>
+          Keep the above selected score
+        </button>
+        <button class="btn" id="roll-dice-btn">Roll the selected dice</button>
       </div>
```

Comment:
Closing tag for the `div` to `form` suggestion above.

```suggestion
      </form>
```